pkg/subctl/benchmark/throughput.go

```diff
@@ -0,0 +1,62 @@
+package benchmark
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartThroughputTests() {
+	gomega.RegisterFailHandler(func(message string, callerSkip ...int) { panic(message) })
+	f := framework.NewBareFramework("throughput")
```

Comment:
nit: The generated namespace for the pods is "e2e-tests-throughput-zpcgt". Since this is not part of e2e-tests, can we just use something like "thoughtput-xxxxx"?

Reply:
This change will have to be first implemented in the Shipyard Repo

Reply:
I think it's OK for it to be "e2e", may be we can improve that later in shipyard too.